src/components/data/AspectRatio.js

```diff
@@ -0,0 +1,81 @@
+import classnames from 'classnames';
+import { cloneElement, toChildArray } from 'preact';
+
+/**
+ * @typedef {import('preact').ComponentChildren} Children
+ *
+ * @typedef AspectRatioProps
+ * @prop {Children} children
+ * @prop {'cover'|'contain'|'fill'|'none'} [objectFit="cover"]
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit mentions a "scale-down" value for object-fit as well. Is that worth including?

Reply:
Yeah, maybe I should be completist here; I was aware of `scale-down` but didn't feel that it would be a typical usage. But who am I to judge? I don't love the feeling of re-implementing `object-fit` via a prop here...I'll reflect. But yeah, I can add `scale-down`.